### ISSUE TITLE
Amazing grate project but need improvement

### ISSUE THREAD

**Issue by Lablnet** [umihico/thumbnailed-portfolio-websites#29]
Hey the design is not good it shout be Mobile responsive

**Comment by umihico** [comment]
Yeah designing must be my next task. It's too ironic if this website gathering cool designs have the worst design. Thank you for pointing "responsive for mobiles"!
